streaming_data_types/array_1d_se00.py

```diff
@@ -150,7 +150,7 @@ def deserialise_se00(buffer: Union[bytearray, bytes]) -> Response:
     return Response(
```

Comment:
Let's move the flagmap and the imports outside of the function.
The imports will only happen once but it will be inefficient to recreate the dictionary every time.